wallet/wallet.go

```diff
@@ -380,14 +371,14 @@ func (sw *SingleAddressWallet) FundTransaction(txn *types.Transaction, amount ty
 		return nil, nil
 	}
 
-	elements, err := sw.store.UnspentSiacoinElements()
+	sw.mu.Lock()
+	defer sw.mu.Unlock()
+
+	elements, _, err := sw.store.UnspentSiacoinElements()
```

Comment:
There's no reason to pass the elements into `selectUTXOs` over calling it within the function now.